### PR TITLE
Anoosha

### PR DESCRIPTION
**Pull request by Anoosha-Sehar**
Genome Tracker terms

### CODE REVIEW

**Review by Anoosha-Sehar — COMMENTED** [review]
**green powder supplement:** Added parent term '`multi-vitamin and multi-mineral supplemen`t' instead of '`botanical supplement with multi-vitamin/mineral`'. As definition of 'green powder supplement' says that it is also made from seaweed, not exactly sure if seaweed comes under the category of plants? It's a green algae, which makes it plant like protist. But I think it's not a true plant.

File: src/ontology/foodon-edit.owl [hunk 1]
```diff
@@ -18870,6 +18877,63 @@ AnnotationAssertion(dc:date obo:FOODON_00004130 "2021-12-14T21:47:32Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00004130 "food mixture"@en)
 SubClassOf(obo:FOODON_00004130 obo:NCIT_C45305)
 
+# Class: obo:FOODON_00004147 (marine protein supplement)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004147 "A dietary supplement to receive the full nutritional benefits from eating whole fish while eliminating the cooking, tasting and chewing."@en)
```
Comment (ddooley) [inline src/ontology/foodon-edit.owl]: I think this could be simplified to "A dietary supplement made from protein derived from marine animals".

**Review by ddooley — CHANGES_REQUESTED** [review]
Hi Anoosha!  Sorry I didn't catch this earlier, but we should add all additives to the FoodOn Robot Tables Dietary Additives tab instead of directly into the foodon-edit.owl file.  I can then rerun the robot command and have those additives appear via import. Ok?

**Comment by Anoosha-Sehar** [comment]
> Hi Anoosha! Sorry I didn't catch this earlier, but we should add all additives to the FoodOn Robot Tables Dietary Additives tab instead of directly into the foodon-edit.owl file. I can then rerun the robot command and have those additives appear via import. Ok?

Sure, no worries! I will add them in Robot file. Then maybe I should revert these commits.

**Comment by Anoosha-Sehar** [comment]
I've added the terms in FoodOn Robot Tables/Dietary Supplement gsheet. Reverting all the commits except one 'new mexico Chile powder'.

**Comment by ddooley** [comment]
Ok, indeed commits would need reverting, or just (oy!) drop pull request branch and just do the chili powder one.

**Pull request merged**